api/server/server.go

```diff
@@ -1062,14 +1062,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(err.Error(), "Timed out") {
```

Comment:
Could make an errors subpackage for state.... or just a generic errors subpackage on daemon.
Testing error values is bad.